tests/kafkatest/services/security/templates/jaas.conf

```diff
@@ -55,6 +55,7 @@ KafkaServer {
     useKeyTab=true
     storeKey=true
     keyTab="/mnt/security/keytab"
+    refreshKrb5Config=true
```

Comment:
Reason to include this change: https://confluentinc.atlassian.net/browse/CPF-392